include/ck/tensor_operation/gpu/thread/threadwise_tensor_slice_transfer.hpp

```diff
@@ -1007,6 +1007,13 @@ struct ThreadwiseTensorSliceTransfer_v4
 
     using SrcCoordStep = decltype(make_tensor_coordinate_step(SrcDesc{}, Index{}));
 
+    static constexpr index_t PackedSize = []() {
+        if constexpr(is_same_v<remove_cvref_t<SrcData>, pk_i4_t>)
+            return 2;
+        else
+            return 1;
+    }();
```

Comment:
We should have type trait to retrieve this information from type.

Reply:
Yes, we can leave it to the next PR. @geyyer